IDT.MoneyTransfer/Main/UI/Pages/MoneyTransfer/TransferAmountForm.cs

```
@@ -22,24 +27,81 @@ public class TransferAmountForm: BasePage
+        private IWebElement TxtReceiving;
+
+
+        public void FillTransferAmountForm(string printingOption)
         {
-            if (printingOption.Equals("Regular"))
+            switch (printingOption)
             {
-                Utils.WaitUntil(5);
-                CommonActions.WaitUntilElementExists("[for= 'notificationInfo_PrintingOption_regular']");
-                CommonActions.ClickWebElement(this.RdoPrintRegular);
+                case Constants.Regular:
+                    new PrinterService().PrinterMethod(new RegularPrinter(), RdoPrintRegular);
+                    break;
+                case Constants.Small:
+                    new PrinterService().PrinterMethod(new SmallPrinter(), RdoPrintSmall);
+                    break;
+                case Constants.NoPrinter:
+                    new PrinterService().PrinterMethod(new NoPrintingOfReceiptPrinter(), RdoNoPrint);
+                    break;
+                default:
+                    throw new Exception("Unknown carrier");
+
             }
-            else if (printingOption.Equals("Small"))
+
+            VariablesGlobals.AmountToReceive = GetTxtReceiving();
+
+        }
+
+        public ReceiptPage ClickBtnProcess()
+        {
+            CommonActions.ClickWebElement(this.BtnProcess);
+            return new ReceiptPage();
+        }
+
+        public string GetTxtReceiving()
+        {
+            return this.TxtReceiving.GetAttribute("value");
+        }
+
+        public double GetTxtSellRate()
+        {
+            double resulSellRate = 0.0;
+            string[] rate = this.TxtSellRate.Text.Split(' ');
+            try
             {
+                resulSellRate = double.Parse(rate[3]);
             }
-            else if (printingOption.Equals("No printing of receipt"))
+            catch (Exception e)
             {
+
+                Console.WriteLine("It can't convert to double", e.Message);
```

If we don't want to that the test continue we should raise an exception and not a Console.WriteLine